editoast/src/views/infra/lines.rs

```diff
@@ -0,0 +1,59 @@
+use crate::error::Result;
+use crate::infra::Infra;
+use crate::infra_cache::{InfraCache, ObjectCache};
+use crate::map::Zone;
+
+use crate::DbPool;
+use actix_web::dev::HttpServiceFactory;
+use actix_web::get;
+use actix_web::web::{block, scope, Data, Json, Path};
+use chashmap::CHashMap;
+use editoast_derive::EditoastError;
+use thiserror::Error;
+
+/// Return `/infra/<infra_id>/lines` routes
+pub fn routes() -> impl HttpServiceFactory {
+    scope("/lines").service(get_line_bbox)
+}
+
+#[derive(Debug, Error, EditoastError)]
+#[editoast_error(base_id = "infra:lines")]
+enum LinesErrors {
+    #[error("no line with code {0} found")]
+    LineNotFound(i32),
```

Comment:
Just a suggestion to add context to your error.

```suggestion
    LineNotFound { line_code: i32 },
```